bucket/scratch.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "3.18.1",
+    "description": "Educational drag-and-drop programming environment",
+    "homepage":  "https://scratch.mit.edu/download",
+    "license":  "CC-BY-SA-2.0",
+    "url":  "https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe#dl.7z",
+    "hash":  "152d7f8aa902e2ca48bd0210e3ccb370d39ce648f51fbe8fdba87e1ee410162b",
+
```

Comment:
```suggestion
    "url": "https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe#dl.7z",
    "hash": "152d7f8aa902e2ca48bd0210e3ccb370d39ce648f51fbe8fdba87e1ee410162b",
```

Reply:
> When was latest version released?

Release page is [here](https://github.com/LLK/scratch-desktop/releases).
The gui itself is BSD ([license](https://github.com/LLK/scratch-desktop/blob/develop/LICENSE)).
But sharing projects on the website is covered by the freeware license.